OpenRA.Mods.Common/Widgets/Logic/Ingame/WorldTooltipLogic.cs

```diff
@@ -80,6 +81,8 @@ public WorldTooltipLogic(Widget widget, World world, TooltipContainerWidget tool
 						}
 				}
 
+				labelText += " " + world.WorldActor.Trait<DomainIndex>().GetDomain(viewport.Cell);
```

Comment:
You can hover over actors to see domain index. This is used to test if clusters are being updated accordingly, the map Minor Pressure can be used as testcase